README.md

```diff
@@ -9,247 +9,128 @@ Also, see the [API documentation](https://aws.github.io/aws-encryption-sdk-c/htm
-other AWS services, you can omit the `-DBUILD_ONLY="kms"` argument, but the build will take much longer.
-
-    git clone -b 1.8.32 https://github.com/aws/aws-sdk-cpp.git
-    mkdir build-aws-sdk-cpp && cd build-aws-sdk-cpp
-    cmake -G Xcode -DBUILD_SHARED_LIBS=ON -DBUILD_ONLY="kms" -DENABLE_UNITY_BUILD=ON ../aws-sdk-cpp 
-    xcodebuild -target install ; cd ..
-
-Now skip to the "Mac: Build and install the AWS Encryption SDK for C" section below.
-
-#### Mac: (Option 2) C only build dependencies
-
-Build and install aws-c-common:
-
-    git clone -b v0.4.42 https://github.com/awslabs/aws-c-common.git
-    mkdir build-aws-c-common && cd build-aws-c-common
-    cmake -G Xcode -DBUILD_SHARED_LIBS=ON ../aws-c-common
-    xcodebuild -target install ; cd ..
-
-#### Mac: Build and install the AWS Encryption SDK for C
-
-Brew installed OpenSSL 1.1 to a place that is not picked up by default so we will
-set the directory manually in our build.
-
-    git clone https://github.com/aws/aws-encryption-sdk-c.git
+    git clone --recurse-submodules https://github.com/aws/aws-encryption-sdk-c.git
     mkdir build-aws-encryption-sdk-c && cd build-aws-encryption-sdk-c
-    cmake -G Xcode -DBUILD_SHARED_LIBS=ON -DOPENSSL_ROOT_DIR="/usr/local/opt/openssl@1.1" ../aws-encryption-sdk-c 
+    cmake -G Xcode -DOPENSSL_ROOT_DIR="/usr/local/opt/openssl@1.1" ../aws-encryption-sdk-c 
     xcodebuild -target install; cd ..
 
-You have successfully built and installed the AWS Encryption SDK for C.
+### Windows Build Recipe
 
-### Building on Windows
+**Note**: See the docker-images folder for some Windows build recipes that automate many of these steps.
 
-**Note**: _see the docker-images folder for some Windows build recipes that automate many of theses steps_
+Start by installing Visual Studio version 15 or later with the Windows Universal C Runtime and [Git for Windows](https://git-scm.com/download/win).
 
-Start by installing Visual Studio version 15 or later with the Windows Universal C Runtime and
-[Git for Windows](https://git-scm.com/download/win).
+Use the "x64 Native Tools Command Prompt" for all commands listed here. Run the following commands in the directory where you want to do the build and installation.
 
+Install Microsoft vcpkg by [following these directions](https://github.com/microsoft/vcpkg#quick-start-windows).
 
-Use the **x64 Native Tools Command Prompt** for all commands listed here. Run the following commands in the
-directory where you want to do the build and installation.
+Use vcpkg to integrate with Visual Studio and to install prerequisites:
 
     mkdir install && mkdir build && cd build
     git clone https://github.com/Microsoft/vcpkg.git
     cd vcpkg && .\bootstrap-vcpkg.bat
     .\vcpkg install curl:x64-windows openssl:x64-windows && cd ..
 
-#### Windows: (Option 1) C and C++ build dependencies
+You may also want to integrate vcpkg and Visual Studio: `.\vcpkg\vcpkg integrate install`
```

Comment:
Does the specified command do the integration? If so:

To integrate vcpkg and Visual Studio (optional), use the following command: `.\vcpkg\vcpkg integrate install`

Reply:
This is one of the things that I was unhappy with and am fixing, so hold this thought please for the next round